package.json

```diff
@@ -84,5 +75,12 @@
         "lint": "eslint --cache clients/web Gruntfile.js grunt_tasks scripts clients/web/test && pug-lint clients/web/src/templates",
         "docs": "grunt docs",
         "esdoc": "esdoc"
+    },
+    "files": [
+        "clients/web/src/"
+    ],
+    "main": "clients/web/src/index.js",
```

Comment:
I think we can remove it.  The outer package shouldn't really be dependency of anything else anyway.  The only reason I can see to publishing it is to provide the built sources, in which case the user will just copy them to a local static resources path.  In which case, "resolving" the main script doesn't make much sense.